### ISSUE TITLE
Change pool ( Zilliqa )

### ISSUE THREAD

**Issue by petko182** [trexminer/T-Rex#363]
Hi, is there command like "--pool_strategy=min_epoch" in teamredminer? With this command you can mine eth on one pool and zilliqa on another one, miner automatically switch pool from mining eth to mining zilliqa. Thx

**Comment by pdieppa** [comment]
I don't believe it is needed with T-Rex. It will do it automatically. I use EZIL (https://ezil.me/?p=871f) and I'm very happy with T-Rex performance.

**Comment by petko182** [comment]
I know, I'm mining zil there ( only with few amd cards because of teamredminer ), but no eth... so I'm curious if there is similar command, then i can use nvidia cards as well :D

**Comment by pdieppa** [comment]
You don't need anything special. I mine zil using T-Rex for my 3060 Ti and 3070 with no Issues. I just specify the coin as ETH and it works. It will change automatically every 110 minutes. Give it a try. I use HiveOS and here is my Flightsheet. Nothing fancy. 

![image](https://user-images.githubusercontent.com/74556152/113345446-75c41580-9300-11eb-97f4-4bb92d28c811.png)

**Comment by pdieppa** [comment]
BTW, I have 5700XTs and a Vii running TRM also.

**Comment by petko182** [comment]
I see, but you mine at one pool. With TRM i can mine ETH in one pool, and when comes time ( mining window ) to mine ZIL all cards switch from mining ETH in one pool to mine for 90s ZIL on different pool and after that time switch back to ETH.

**Comment by petko182** [comment]
<img width="806" alt="Snímka obrazovky 2021-04-01 o 22 00 43" src="https://user-images.githubusercontent.com/81773031/113347791-f5b8a280-9335-11eb-9fb6-ea04109314e0.png">

**Comment by pdieppa** [comment]
I don't know if EZIL allows that type of split connection. I do both ETH and ZIL at EZIL.

**Comment by petko182** [comment]
As you see, its possible :) For now i found solution just for TRM, not for trex sadly ( i have 0,6 Ghs rig running trex )

**Comment by pdieppa** [comment]
If you like ZIL, there is a way you can mine ZIL all day long. You mine ETH and they give you the equivalent in ZIL. It's good for mining coins that are unmineable. According to the website, based on my ETH hashrate, I can generate just under 400 ZIL a day @ 800mh

**Comment by petko182** [comment]
How? I thought its possible just limited time per day... not like "ordinary" mining like ETH, RVN, etc., am I wrong?

**Comment by petko182** [comment]
Understand now... so basically you mine ETH but payout is in ZIL. But whats the "ratio", you can change ETH in exchange.... but yop taxes :D

**Comment by pdieppa** [comment]
https://unmineable.com/?ref=smbq-uhjn
